pennylane/transforms/unitary_to_rot.py

```diff
@@ -78,9 +85,14 @@ def qfunc():
                 zyz_decomposition(op.parameters[0], op.wires[0])
             # Two-qubit unitary operations
             elif qml.math.shape(op.parameters[0]) == (4, 4):
-                decomp = qml.transforms.invisible(two_qubit_decomposition)(
-                    op.parameters[0], op.wires
-                )
+                # Can only decompose if they require no CNOTs, or 3 CNOTs currently
+                try:
+                    decomp = qml.transforms.invisible(two_qubit_decomposition)(
+                        op.parameters[0], op.wires
+                    )
+                except NotImplementedError:
+                    # Keep the operation as-is
+                    decomp = [op]
```

Comment:
Rather than do nothing, maybe it makes more sense to do one of the following:

- Raise a `UserWarning` ('one or more `QubitUnitary`'s were not decomposed because...')
- Raise an exception?